rules/no-identical-title.js

```diff
@@ -7,6 +7,19 @@ function purify(node) {
 	return node && espurify(node);
 }
 
+function isStaticTemplateLiteral(node) {
+	return node.expressions.reduce(function (res, expr) {
+		return res && isStatic(expr);
+	}, true);
```

Comment:
Man, didn't even know `every` and `some` were built-in array methods. You learn everyday.